calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -7475,8 +7489,8 @@ protected MergedList loadChannels(String siteId, String primaryCalendarReference
 	 * Get the user agent we should use for request to get other calendars.
 	 * @return The user agent.
 	 */
-	String getUserAgent() {
-		return "Sakai/"+ m_serverConfigurationService.getString("version.sakai", "?") + " (Calendar Subscription)";
+	String getUserAgent(String context) {
+		return "Sakai/"+ m_serverConfigurationService.getString("version.sakai", "?") + "/" + context + " (Calendar Subscription)";
```

Comment:
This will leak contexts (Site IDs) into the logs of other services, I don't think this is a problem but thought I should flag it.